ibm/service/power/data_source_ibm_pi_volume_group.go

```diff
@@ -0,0 +1,126 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	//"fmt"
+
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+)
+
+func DataSourceIBMPIVolumeGroup() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPIVolumeGroupRead,
+		Schema: map[string]*schema.Schema{
+			PIVolumeGroupName: {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Name of the volume group",
+				ValidateFunc: validation.NoZeroValues,
+			},
+			helpers.PICloudInstanceId: {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			// Computed Attributes
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"replication_status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"consistency_group_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status_description": {
+				Type:     schema.TypeList,
```

Comment:
If I am not wrong status_description is not a list. It is plain map everywhere else.

Reply:
In map `Elem` doesn't support resource type schema and interface{}

Reply:
refactored the status description attribute `status_description_erros`